### PR TITLE
Show "Get it on GPlay"

### LINKED ISSUE #319: Beta Testing on GPlay -> Get it on Google Play
get.delta.chat still speaks about "Beta Testing on Google Play". This is not necessary anymore.

### PR DESCRIPTION
**Pull request by ghost**
closes #319

### CODE REVIEW

File: _data/lang/en.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ en:
     beta_on: Beta on
     dl_fdroid: Get it on F-Droid
     beta_gplay: Beta testing on Google Play
```
Comment (Simon-Laux) [inline _data/lang/en.yaml]: you don't removed the old string?

File: _includes/download-boxes.html [hunk 2]
```diff
@@ -68,7 +68,7 @@ <h1>{%if tx.download.other != ""%}{{ tx.download.other }}{%else%}{{ txEn.downloa
                     <img src="../assets/badges/get-it-on-fdroid.png" alt="{%if tx.download.dl_fdroid != ""%}{{ tx.download.dl_fdroid }}{%else%}{{ txEn.download.dl_fdroid }}{%endif%}" />
                 </a>
                 <a href="https://play.google.com/store/apps/details?id=chat.delta" class="img-badge">
-                    <img src="../assets/badges/get-it-on-gplay-beta.png" alt="{%if tx.download.beta_gplay != ""%}{{ tx.download.beta_gplay }}{%else%}{{ txEn.download.beta_gplay }}{%endif%}"  />
+                    <img src="../assets/badges/get-it-on-gplay.png" alt="{%if tx.download.gplay != ""%}{{ tx.download.gplay }}{%else%}{{ txEn.download.gplay }}{%endif%}"  />
```
Comment (Simon-Laux) [inline _includes/download-boxes.html]: you are referencing the file you deleted?

File: _data/lang/en.yaml (hunk 1 above)
Comment (Simon-Laux) [reply _data/lang/en.yaml]: do you plan on re-purposing it for a beta testing button?

Comment (ghost) [reply _data/lang/en.yaml]: Good point.

Comment (ghost) [reply _data/lang/en.yaml]: I think I just wasn't sure what would happen, but of course, it isn't used anywhere else, so we can drop it.

**Comment by ghost** [comment]
Oh, and it seems I deleted the wrong image. >.<

**Comment by r10s** [comment]
> Oh, and it seems I deleted the wrong image. >.<

also wanted to say that :)

but i would not delete the beta-image at all, maybe we need it at some point again. i think it is complicated to get that from some old commit and compared to all the other screenshots and stuff, this does not add much.

**Review by r10s — APPROVED** [review]
lgtm now :)

**Comment by ghost** [comment]
Well, it's additional traffic every time we deploy the page. But doesn't make a huge difference I guess. I reset the head and excluded the commit from this PR.

**Comment by Simon-Laux** [comment]
that additional traffic makes not the biggest impact, there are other places we can save more.

**Pull request merged**